### PR TITLE
Add example commands/scripts for hitting the endpoint

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

File: docs/source/tutorial.rst [hunk 1]
```diff
@@ -68,10 +68,40 @@ If you want to run only Ansible playbook, you can just run ``ansible-playbook -i
...
+
+With the above command, you can hit the API via ``http://localhost:8080``. You can directly hit the endpoint via `curl`
+or other commands, or you can use `the OpenAI Python library <https://github.com/openai/openai-python>`_.
+
+Here is an example command for listing all available models and hitting the chat endpoint.
+
+.. code-block:: console
+
+   curl http://localhost:8080/v1/models | jq
+   curl --request POST "http://localhost/v1/chat/completions" -d '{"model": "google-gemma-2b-it-q4", "messages": [{"role": "user", "content": "What is k8s?"}]}'
```
Comment (Ladicle) [inline docs/source/tutorial.rst:85]:
```suggestion
   curl --request POST "http://localhost:8080/v1/chat/completions" -d '{"model": "google-gemma-2b-it-q4", "messages": [{"role": "user", "content": "What is k8s?"}]}'
```

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -68,10 +68,40 @@ If you want to run only Ansible playbook, you can just run ``ansible-playbook -i
...
+
+Here is an example Python code for hitting the chat endpoint.
+
+.. code-block:: python
+
+   from openai import OpenAI
+
+   client = OpenAI(
+     base_url="http://localhost:80/v1",
```
Comment (Ladicle) [inline docs/source/tutorial.rst:94]:
```suggestion
     base_url="http://localhost:8080/v1",
```

**Review by guangrui-cloudnatix — APPROVED** [review]

File: docs/source/tutorial.rst [hunk 3]
```diff
@@ -68,10 +68,40 @@ If you want to run only Ansible playbook, you can just run ``ansible-playbook -i
 Step 4: Interact with the LLM Service
 -------------------------------------
 
+You can access the API endpoint by establishing a SSH port-forwarding.
+
 .. note::
 
-   TODO: Fill this out.
+   ansible all -i inventory.ini --ssh-extra-args="-L8080:localhost:80" -m shell -a "sleep infinity"
```
Comment (Ladicle) [inline docs/source/tutorial.rst:75]: Just checking, can we access kind's(docker-engine) NodePort from localhost?

Comment (kkaneda) [reply docs/source/tutorial.rst:75]: Yes, it worked fine with my testing.

**Review by Ladicle — APPROVED** [review]

File: docs/source/tutorial.rst (hunk 3 above)
Comment (kkaneda) [reply docs/source/tutorial.rst:75]: I could be wrong, but `extraPortMappings` (https://kind.sigs.k8s.io/docs/user/configuration/#extra-port-mappings) is providing the mapping.

https://github.com/llm-operator/llm-operator/blob/main/aws/scripts/setup-kind-cluster.sh#L16

```
ubuntu@ip-172-31-29-15:~$ docker ps
CONTAINER ID   IMAGE                  COMMAND                  CREATED       STATUS         PORTS                                                                 NAMES
c74eba97a815   kindest/node:v1.29.2   "/usr/local/bin/entr…"   6 hours ago   Up 3 minutes                                                                         nvkind-7l7fk-worker
2c8e74526ca4   kindest/node:v1.29.2   "/usr/local/bin/entr…"   6 hours ago   Up 3 minutes   0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 127.0.0.1:33255->6443/tcp   nvkind-7l7fk-control-plane
```

**Pull request merged**